.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -11,7 +11,7 @@ https://github.com/Kong/kong/blob/master/CONTRIBUTING.md#contributing
 ### Checklist
 
 - [ ] The Pull Request has tests
-- [ ] There's an entry in the CHANGELOG
+- [ ] A changelog file has been added to `CHANGELOG/unreleased/kong` or adding `skip-changelog` label on PR if unnecessary. [CHANGELOG/README.md](https://github.com/Kong/kong/CHANGELOG/README.md)
```

Comment:
```suggestion
- [ ] A changelog file has been added to `CHANGELOG/unreleased/kong` (You can ask @vm-001 to do it for you)
   /or/
- [ ] The `skip-changelog` label  has been added because no changelog entry is required
```